### PR TITLE
Rename main app

### PR DESCRIPTION
**Pull request by kosmos342**
Use our standard name for main app

### CODE REVIEW

File: yolapi/main/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from main.celery_setup import app as local_celery_app
```
Comment (snitch) [inline yolapi/main/__init__.py:1]: F401 'main.celery_setup.app as local_celery_app' imported but unused

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by RayeN — APPROVED** [review]

**Pull request merged**